src/main/java/domain/LastWeekLottoNumber.java

```diff
@@ -0,0 +1,34 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LastWeekLottoNumber {
+
+    private static final String SPLIT_STRING_DELIMITER = ",";
+    private final List<Integer> lastWeekLottoNumber;
+    private final String inputLastWeekLottoNumber;
```

Comment:
```suggestion
    private static final String SPLIT_STRING_DELIMITER = ",";
    
    private final List<Integer> lastWeekLottoNumber;
    private final String inputLastWeekLottoNumber;
```

아주 현실적인 클래스명이네요 ㅋㅋㅋㅋ 좋습니다!

Reply:
ㅋㅋㅋㅋㅋ감사합니다!!